src/Connection.php

```diff
@@ -420,6 +420,24 @@ public function prepareBindings(array $bindings)
         return $bindings;
     }
 
+    /**
+     * @inheritDoc
+     */
+    protected function escapeBool($value)
+    {
+        return $value ? 'true' : 'false';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function escapeString($value)
+    {
+        return str_contains($value, "\n")
+            ? 'r"""' . addcslashes($value, '"') . '"""'
+            : '"' . addcslashes($value, '"') . '"';
```

Comment:
probably should escape slashes

```suggestion
            : '"' . addcslashes($value, '"', '\') . '"';
```